docs/userguides/plugins.md

```diff
@@ -118,3 +118,36 @@ CLI arguments are decoded and passed in to the application with `click`.
 CLI plugins will use the plugin registration process defined above. The
 CLI plugins should use the `click` library in order to be able to supply
 arguments from the CLI.
+
+## Logging Ape
+
+Ape comes equiped with additional logging tools to help view output. Custom plugins may need to stop a process and log the info. The `Abort` and `ape_cli_context` are available on the `from ape.cli` and _@ape_cli_context decorator_.
+
+
+* **Abort:**
```

Comment:
We should use the `###` instead of bold here (3rd level header).

However, I am not sure `Abort()` makes sense in the Logging part of this guide

Reply:
I will abort Abort.